workspaces/acs/plugins-list.yaml

```
@@ -0,0 +1 @@
+plugins/acs
```

```suggestion
plugins/acs:
```

Since this is expected to a `yaml`, with the optional value at the right being additional arguments for the export-dynamic command.

This doesn't prevent the export though, since the script isn't strict about this.